### PR TITLE
 [gn] Add flutter_driver build file.

### PR DESCRIPTION
**Pull request by awdavies**

### CODE REVIEW

File: packages/flutter_driver/BUILD.gn [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Flutter Authors. All rights reserved.
```
Comment (tvolkert) [inline packages/flutter_driver/BUILD.gn]: I think we still need to use "The Chromium Authors" (/cc @sethladd dyk?).

If that's the case, can you send a separate PR to update the license headers in `fuchsia_remote_debug_protocol`?

**Review by tvolkert — APPROVED** [review]

File: packages/flutter_driver/BUILD.gn (hunk 1 above)
Comment (awdavies) [reply packages/flutter_driver/BUILD.gn]: Double-checked and it looks to be the case from a previous code review.  I'll update frdb
